src/generate_diagnosis.py

```diff
@@ -0,0 +1,104 @@
+from pathlib import Path
+import logging
+
+import pandas as pd
+from rdflib import Graph, RDFS, Literal
+import tqdm
+
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger("generate_diagnosis")
+
+
+def load_concept_relationships(filepath):
+    log.info(f"Loading the {filepath}")
+    df = pd.read_csv(filepath, sep='\t', usecols=['concept_id_1', 'concept_id_2', 'relationship_id'])
+    return df[df['relationship_id'] == "Is a"]
+
+
+def create_graph(df):
+    log.info("Creating the graph")
+    graph = Graph()
+    for index, row in tqdm.tqdm(df.iterrows(), total=df.shape[0], desc="Creating graph"):
```

Comment:
this is super cool! maybe we could consider reusing this for this issue:
- https://github.com/neurobagel/bagel-cli/issues/189

Reply:
yeah, I like this thing for slow tasks.